### PR TITLE
chore: Add posibility to use --auto-skip with earthly

### PR DESCRIPTION
**Pull request by flemzord**

### CODE REVIEW

File: components/Earthfile [hunk 1]
```diff
@@ -1,8 +1,6 @@
 VERSION 0.8
 
-ARG core=github.com/formancehq/earthly
-
-IMPORT $core AS core
+IMPORT github.com/formancehq/earthly:baaa0352c718937be6f9c40fe40879642bbd6132 AS core
```
Comment (Dav-14) [inline components/Earthfile:3]: On avait pas dit qu'on prenait main tout le temps ?

Comment (gfyrag) [inline components/Earthfile:3]: Use a version?

File: ee/Earthfile [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
-
-IMPORT $core AS core
+IMPORT github.com/formancehq/earthly:baaa0352c718937be6f9c40fe40879642bbd6132 AS core
 IMPORT .. AS stack
 
 deploy:
-    FROM core+base-image
-    ARG --required components
-    BUILD --pass-args ./$components+deploy
+  FROM core+base-image
```
Comment (gfyrag) [inline ee/Earthfile:7]: Fix your indent everywhere please.
All the files should have 4 spaces as tab.

File: ee/Earthfile [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
-    FROM core+base-image
-    ARG --required components
-    BUILD --pass-args ./$components+deploy-staging
\ No newline at end of file
+  FROM core+base-image
+  ARG --required components
+  BUILD --pass-args ./$components+deploy-staging
+    
+run:
+  LOCALLY
```
Comment (gfyrag) [inline ee/Earthfile:17]: I think I have seen earthly allow `BUILD ./*+$TARGET`, can you check?

Comment (flemzord) [reply ee/Earthfile:17]: This is not possible because it is not deterministic.

**Review by gfyrag — APPROVED** [review]

**Pull request merged**